dashboard/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "rimraf dist && cross-env NODE_ENV=production webpack --config webpack.config.js",
     "build-debug": "rimraf dist && cross-env NODE_ENV=development webpack --config webpack.config.js",
-    "watch": "cross-env NODE_ENV=development webpack-dev-server",
+    "watch": "cross-env NODE_ENV=development webpack-dev-server --disable-host-check",
```

Comment:
Probably not needed in this PR

Reply:
Huh, normally i make sure that isnt included in my PRs 🤔